swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/mocking.md

```diff
@@ -149,11 +145,11 @@ function getCategoriesResponse() returns string[] {
 }
 ```
 
-#### Stub to return a specific value
+##### Stub to return a specific value
 
 ***main_test.bal***
- 
-This test stubs the behaviour of the `get` function to return a specific value in 2 ways:
+
+This test stubs the behaviour of the `get` method to return a specific value in 2 ways:
```

Comment:
This should be behavior. Check all the places on this page, as it seems it is appearing wrong in many places.